display-composable-mapview/README.md

```diff
@@ -1,4 +1,4 @@
-# Display Composable MapView
+# Display composable mapview
```

Comment:
Let's not use lowercase here, I realize this is outside of the conversion used in the sample names. Both `Composable` and `MapView` are product-API names and should be okay to use for this sample:  

 
```suggestion
# Display Composable MapView
```